Update changelog.md (#50)
MewsSystems/gitbook-channel-manager-api: changelog.md

# Changelog

## 26th March 2020 12:00 UTC

[Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group). So net pricing is supported in reservations also in inventory.

> ## 19th Match 2019 22:00 UTC

Author: ```suggestion
## 19th March 2019 22:00 UTC
```

# Changelog

## 26th March 2020 12:00 UTC

> [Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group). So net pricing is supported in reservations also in inventory.

Reviewer: ```suggestion
[Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group) and [Price](channel-manager-api.md#price) extended in [Update Prices](channel-manager-api.md#update-prices). Net pricing is now supported for inventory upload and reservation delivery.
```

> 

Reviewer 2: Maybe rephrase it to "Net pricing is supported in processGroup and inventory"

> ## 19th Match 2019 22:00 UTC

Author: ```suggestion
## 26th March 2020 12:00 UTC
```
